components/authz-service/storage/postgres/migration/sql/69_add_status_to_get_project.up.sql

```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION
+  query_project(
+    _project_id TEXT,
+    _projects_filter TEXT[]
+  )
+  RETURNS json AS $$
+declare
+  project_json json;
+
+BEGIN
+  WITH staged AS (
+    SELECT r.project_id, count(r.id) AS count
```

Comment:
do you need the count, here? Seems like `staged` would still be null without it.

Reply:
Ah maybe, lemme try.

Reply:
Hmm tried that and it breaks the tests didn't dive deep. Can if we have a strong preference against `count` but that's how ListProjects works currently.

Reply:
Ok cool, nbd. Thanks for looking.

Reply:
Good point, @blakestier, as we are not really using the count.
With this change all the tests still pass:
```
 WITH staged AS (
    SELECT r.project_id
      FROM iam_staged_project_rules AS r
      WHERE project_db_id(_project_id) = r.project_id
      GROUP BY r.project_id
  ),
  applied AS (
    SELECT  r.project_id
      FROM iam_project_rules AS r
      WHERE project_db_id(_project_id) = r.project_id
    GROUP BY r.project_id
  )
```
then change the `case` statement (L31-34) to this:
```
CASE WHEN staged.project_id IS NULL AND applied.project_id IS NULL THEN 'no-rules'
      WHEN staged.project_id IS NULL THEN 'applied'
      ELSE 'edits-pending'
    END
```